lib/membrane_audio_mixer_bin.ex

```diff
@@ -90,123 +63,123 @@ defmodule Membrane.AudioMixerBin do
     availability: :always,
     caps: Raw
 
+  defmodule MixerOptions do
+    @moduledoc """
+    Structure representing options that would be passed to each created Membrane.AudioMixer element.
+    """
+    defstruct caps: nil, frames_per_buffer: 2048, prevent_clipping: true
+
+    @type t :: %__MODULE__{
+            caps: Raw.t(),
+            frames_per_buffer: pos_integer(),
+            prevent_clipping: boolean()
+          }
+  end
```

Comment:
There's no need to create a new struct, you can simply use `%Membrane.AudioMixer{}`